pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -326,7 +326,8 @@ class Build {
                     writeMetadata(versionData)
                     context.archiveArtifacts artifacts: "workspace/target/*"
                 } catch (e) {
-                    context.println("Failed to build installer ${buildConfig.TARGET_OS} ${e}")
+                    context.println("Failed to build ${buildConfig.TARGET_OS} installer")
```

Comment:
Why has `${e}` been removed from this - won't that run a risk of hiding the underlying error from the log?